ohsomeTools/common/client.py

```diff
@@ -141,11 +137,18 @@ def request(
             body = post_json
             requests_method = "POST"
 
+        if body and "bcircles" in body.keys() and len(body["bcircles"]) > 5:
+            split = body["bcircles"].split("|")[0:2]
+            b = body.copy()
+            b["bcircles"] = f'{"|".join(split[0:2])}|[...]'
+        else:
+            b = body
+
         logger.log(
             "url: {}\nParameters: {}".format(
                 self.url,
                 # final_requests_kwargs
-                json.dumps(body, indent=2),
+                json.dumps(b, indent=2),
             ),
             0,
         )
```

Comment:
Does it make sense to log the request here?
Couldn't that be done in the nam.request-method, independent of the invocation?

Reply:
Good point. I'll change it.